src/backbone/resnet_fpn.jl

```diff
@@ -0,0 +1,88 @@
+function conv1x1(in_planes, out_planes, stride=1)
+  return Conv((1,1), in_planes => out_planes, stride=stride, padding=0, bias=False)
+    # """1x1 convolution without padding."""
+end
+
+function conv3x3(in_planes, out_planes, stride=1)
+  return Conv((3,3), in_planes => out_planes, stride=stride, padding=0, bias=False)
+    # """3x3 convolution without padding."""
+end
+
+function BasicBlock(in_planes, out_planes,stride=1,x::AbstractArray)
+  conv1 = conv3x3(in_planes, out_planes, stride)
+  conv2 = conv3x3(out_planes,out_planes,stride)
+  bn1 = BatchNorm(out_planes)
+  bn2 = BatchNorm(out_planes)
+  y = x
+  if (stride == 1)
+    downsample = None
+  else
+    downsample = Chain(
+      conv1x1(in_planes, out_planes, stride),
+      BatchNorm(out_planes)
+    )
+  end
+  y = relu(bn1(conv1(y)))
```

Comment:
The layers already accept activations, so why not have the users select what activation they want and construct the appropriate layer? That is both cleaner, less ahssle for the code base and configurable

Reply:
Of course this applies to all usages of `relu` etc in the models